box: moves mechanics to science

SonixApache opened the pull request:
This PR moves the mechanics workshop to science. 
Maybe it'll make mechanics work together with scientists better. Maybe it won't!
The doors between the workshop and R&D are science-access, the two leading into the science hallway are still standard access. Not sure if I should change that.
To make space for all this:
 - Toxins has been moved to the outpost
 - RD's office has been moved to where toxins was (and the rest filled with maint)
 - Toxins storage was also moved, and the empty space filled with a break room and a small storage room
 - R&D moved to the RD's office.
 - Mechanics workshop moved to where R&D was.
 - Vacant Mechanics workshop converted to construction area.

New science wing:
![sciplusmechupdated](https://user-images.githubusercontent.com/8468269/46930528-712f1800-d046-11e8-91d4-76d8db737b08.png)
New outpost (FUCK TUBES):
![outpost](https://user-images.githubusercontent.com/8468269/46929135-4c36a700-d03e-11e8-999b-b1d91ad2bcce.png)
Vacated mechanics office:
![mechanicsupdated](https://user-images.githubusercontent.com/8468269/46930725-9a9c7380-d047-11e8-9456-dbbf7734aa10.png)


🆑 
 - experimental: Box: Moved the mechanics workshop to the Science wing.
 - experimental: Box: Moved toxins to the science outpost.
 - rscadd: Box: Added a break room to the Science wing. Thanks to slimeboi for styling it with me.

Caboozles commented:
Putting another R&D setup next to R&D seems pretty galaxybrain. What's even the point of it being there at that stage?

SonixApache commented:
I figured being next to eachother so you can share resources / ideas but having your own lathe and printer to still be able to queue up your own shit would be beneficial

same as why robotics has a circuit printer

otatoh commented:
New toxins and the breakroom are great, although imo it would make more sense to leave r&d in the place it was, and the mechanics lab in the back.

While I understand you want to leave a place for people to build, the old mechanics office is pretty barren, and I worry it'll be abandoned office 2.0. I suggest moving the APC/air alarm to the back, adding some windows/a counter to the front, and making the door have no access requirement to encourage people to use it.

Shadowmech88 commented:
>Maybe it'll make mechanics work together with scientists better.

Wasn't the original purpose of placing mechanics away from Science specifically to encourage them to interact more with the rest of the station? R&D has always had most of the potential that Mechanics has, but they never did the things Mechanics were supposed to do because Science was their fortress, and they'd rather make toys for themselves than leave to go put new components into everything.
If Mechanics is placed inside Science, literally exactly where R&D used to be, isn't there a large risk of them falling victim to the same problem that led to their creation?

That said, I actually think moving Toxins to the asteroid is a good idea. Not only will it cut down on station-destroying accidents, hilarious though they may be, it may also encourage more cooperation with the miners since they can just knock on the Research Outpost door while they're already on the asteroid.

SonixApache commented:
Having access to materials in one place might give them the stuff they need to actually *do* things, and the things they *do* is go out and work on stuff. I dunno, don't think they've got the space to really fortress up.

Irradiation commented:
@Shadowmech88 In current practice, Scientists are the ones that end up upgrading machines because they get the mats for better components. That and mechanics have a tendancy to fuck off somewhere (usually to space) due to lol no responsibilities. This ends up in people breaking into mechanics to scan machines & print their own machines.

SonixApache commented:
- beautified science a bit
 - removed rogue pipes
 - did the thing for the vacant office

SonixApache commented:
- fixed all the broken and missing pipes I could find (again, because DM ate my changes)
 - added cameras here and there
 - break room is now actually powered and stuff (thank you slimes)
 - removed some rogue floating shit

this is 12 hours of mapping straight, i hope everything's set now

Kyunny commented:
*inhales*
Move mechanics to cargo :^)

CptWad commented:
needs a toxins bomb dectector computer on the main zlevel
They are locked to specific zlevels and I think pick up meteor showers on their zlevel only
you also cannot have player spawns on the asteroid because of blob and rev

Riekland commented:
Stunning & Brave

SonixApache commented:
there's a bhangometer on the box bridge
blob and rev don't exist anyway, but noted

aceedex commented:
I dont like it under the principle that two RnD getups near each other are grug tier and now very susceptible to bomb two for ones.
Ive always said that spacepods is rightful mechanic clay and there's more than enough room in the spacepod bay to move em there if thats what needs be done

soyconsumer commented:
![ebin](https://user-images.githubusercontent.com/36347086/46993647-8a040000-d110-11e8-901d-4de7f267749b.png)

I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE commented:
This will 100% turn the Mechanic's office into Science property
I'm excited to see the feuding in action

jknpj commented:
I'm just merging all mapping changes today so here we go

Shadowmech88 commented:
Seems like a bit of a large change to be merged in a day, if you ask me.

Irradiation commented:
I only think about it now because somebody pointed that out, but CEs can't even go into the mechanic workshop because you need science access to get there without breaking walls. So much for an advanced device analyzer :smirk:

Irradiation commented:
Addendum: now that mechanic machines are entirely science property, you can't open a single door to operate the mechanic machines that nobody will care about if you're there. I hereby welcome our new possessive scientists overlords that may or may not throw a fit if they see you in the room using the machines. Or they might not care and <s> just give you the supplies/machines/toys you asked for</s>.

Caboozles commented:
My opinion on this is still strongly in the camp of put mechanics in engineering, remove the glubbs from Mechanics' and let them make their way to engineeirng or tcomms if they want them, and remove science access from them, but on a positive note, at least we vaguely got materials, even if feasibly absolutely nothing changes. There was no cooperation, and instead people would constantly push their way in to take materials from the mechanics' protolathe and steal device analyzers to scan their own stuff to flatpack.

Also, the disposal in mechanics just dumps in R&D, to add insult to injury.

morenaaa commented:
I'M HERE TO COMPLAIN

Mechanics is not a part of Science and was never supposed to be part of Science. It's an engineering sub-department and that's how it should have stayed.
The move away from Plastic being needed for tier3 components and onto regular mining materials is what caused this confusing in the first place. Science complained boohoo shit about not being able to make tier3 with all their miner spoils when they weren't supposed to be able to in the first place.
Mechanics is an extension of Engineering. They relate to the department just like Atmospheric Technicians do. While Engineers work with the stations power, maintaining the wire-net and keeping everything turned on it, Atmospheric Techs work on the pipe-net, keeping the station pressurized and all the bad gas out of the air, and Mechanics work on another part of the station: The machines. A mechanics job is to replace, maintain and upgrade machinery on the station. It's obvious from the kit they're given that this is what they're supposed to be doing, but you gave into pressure of science cry babies when they realised they didn't have so many toys any more.
Give us back plastic for Tier 3 parts, get rid of this science domination on the game. There was nothing wrong with it in the first place.
Put Mechanics back where it used to be. It's not part of the Science Department and it doesn't belong.


plus, jesus CHRIST is the new layout ugly. Why even have two near identical set ups 4 tiles from each other? You might as well have just merged Mechanics into Science and removed the job with what you did.
You REMOVED TOXINS FROM THE MAINSTATION.
WHY. Now there are no TTVs on the main station and if you get them as an objective you have to try and get onto the outpost to get them. This is ridiculous. 
You've also just massively buffed science antags making bombs because now, if they want to use the set up, they are going to be off station and almost no one has a chance of catching them even working it.
This went through insanely fast for such a massive change, it's honestly disgusting. It's almost like you don't play any of these jobs.

Irradiation commented:
>You REMOVED TOXINS FROM THE MAINSTATION.
WHY. Now there are no TTVs on the main station and if you get them as an objective you have to try and get onto the outpost to get them. This is ridiculous.

Already a thing on Bagel for all it's worth, but I don't hear people complaining (People complain more about the map itself than the outpost things).

Hoshi-chan commented:
Make the mechanics office part of Supply, and park them next to cargo. Cargonia will welcome the addition of R&D machines, and mechanics will finally get first dibs on materials from the asteroid.

Everyone wins.

SonixApache commented:
@morenaaa
>Mechanics are engineering, not science

Mechanics are quite literally the fusion between them; they were born from what science is (was) supposed to do (Build, replace, upgrade & repair machines), and fill the niche of telecomms specialists (since those are machines). They are in essence just an alt-title for scientist with different access, much like roboticists.

>plastic parts!

I dunno about that. When it was silvergoldetc, people wanted plastic. When it was plastic, people wanted silvergoldetc. Maybe we should just make a seperate entry for T3 parts made outta plastic?

>layout sucks

I seriously think having two lathes (and thus, two queues) is a good thing. 
I also realize that the two departments might compete for table space (especially since the bell is set to ring scientists), but I don't see that as a necessarily bad thing. If you think there's a better way to move stuff around, do post it.
I've been thinking about axing the two-door access airlock, to make some more room for table space / a wait-for-requests room.

>toxins shouldn't be on the outpost!

toxins was a mistake, don't @ me about this

>This went through insanely fast for such a massive change, it's honestly disgusting.

I agree that it went through way too fast. I was expecting it to be up for a week or two.

Caboozles commented:
I totally disagree about mechanics being science. Science in name only at best.
Prior to this change they only had tsci access, and ONLY on box. Science access provided no functional purpose.

On the other hand, while they have far more access to engineering, as the foyer holds lockers, parts, vends, materials, and even canisters if someone opened secure storage. There’s a lot more meat there, and instead of trying to focus mechanics on engineering (IE remove glubbs from the mechanics office and throw their locker into tcomms), you’ve centered it around the area they consider their mortal enemies.

morenaaa commented:
Sonix I'm sorry but you're just wrong about Mechanics being anything to do with Science, let along calling them an alt title to Scientist.
Science has nothing to do with telecomms (A big thing some mechanics love doing, they pick the role specifically to play around with telecomms). It's an engineering thing. Like Caboozles says, They've always, always had more access and use around in engineering than they have in Science. 
But just because it was a job Science was _supposed_ to do, doesn't mean now that the job has been separated it falls into the Science domain. While without the job, yes it falls to Science to upgrade and replace machines, this is only because Engineering don't have access to Protolathes, Circuit Printers, etc. It's just a logic thing that maintenance of the station falls to Engineering, not the Research portion of the station.

And Bagel is a shit map, I do not play it because it is so extremely large and annoying to navigate so I can't exactly comment on it. I think I played one round on it as a Mechanic traitor in which I used the fact my office was disconnected from anywhere with higher traffic to do some shady things. But if Toxins is on the Outpost, it's not good, it's not great, it's lame.
Toxins is sort of a mistake though. Bombs are predictable and easy to produce, but that doesn't mean anything when esword ebow is predictable and easy to produce but we don't slowly nerf the combo out because we don't like when people do it. I don't want to turn this into a toxins rant so I'll just say that I still believe it should remain on the main station, especially if TTVs are going to be an antag objective item.

Also I very much don't like the layout, because it increases traffic through your work area and that gets annoying. Mechanics has a lot of big, bulky objects that it moves around and needs to store, if you have scientists and whoever coming and going constantly, moving your stuff around will get annoying. It's also a huge pain in the ass for Scientists now to man the front desk, because they constantly have to run away get things from their little area when people ask for it. Most of the time when you had a scientist, because they were in their little r&d hole, you wouldn't have to bother them with a bell ring and could just ask nicely for what you want and boom, a few seconds later its on the desk. Seeing as mechanics are a lesser played role, and more likely to not be in the office, you've pretty much added an always waiting time to the science front desk, which I'm sure I'm not the only one who runs to a lot to ask for things like welding goggles, stock parts, new boards, when building my own things, etc. And sure, I should be asking the Mechanic for the latter things, but they are rarely played, I don't think they even had a bell at their old office, and even then are rarely in their office. It's a busy job, if you are doing it actively. 
But I still don't understand why two queues is even a necessary thing. Upgraded protolathes are fast enough, it means you'd have to split materials down the middle and manage two pools, and I doubt there's ever a time where you need to be mass producing anything that one protolathe couldn't handle. It's honestly just a waste of space to have all this stuff doubled up within a few tiles of each other, and I stand by that if this is a change people support then we should just remove the Mechanics job, move back to the old Science layout but just add the mechanics machines to a corner in that R&D room.

SonixApache commented:
![oneroomtwojobs](https://user-images.githubusercontent.com/8468269/47186556-36bcc800-d331-11e8-8e46-96615515c993.png)
that would look something like this, and I'm not sure that's an improvement at all.

RonPaul08 commented:
Merge it. Doing god’s work sonix.
